### DISCUSSION TITLE
[img2img] on manual save: WARNING  Saving RGBA image as JPEG: Alpha channel will be lost

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#1391] [Q&A] [answered]
Hi,

I use img2img with a jpg as input (jpegs have no transparency). When saving the result manually, I get

`WARNING  Saving RGBA image as JPEG: Alpha channel will be lost`

in the log. That's new, I think, and shouldn't be logged? Can the output actually contain transparency at all?

**Comment by vladmandic** [comment, marked as answer]
warning comes from a global image handler, there is no save implemented inside each function.
and global handler doesn't know what you wanted to start with, it just knows that it received RGBA image and desired format is JPG, so it prints the message. nothing is broken, image gets converted to RGB and saved.

and yes, some img2img operations result in RGBA. I could transparently convert them or I could remove the warning, but i think this is more accurate.
